index.html

```diff
@@ -9,21 +9,57 @@
-<p>Even though my family didn't have a lot of money when I was growing up, I never gave up on my dream of becoming a software engineer because I love working with technology. I recently moved to the United States, embarking on a new chapter of exploration and learning.</p>
-<p>Family is super important to me, and I cherish every moment I spend with them. I also really enjoy being outside, whether it's taking a walk in the park or having a picnic on the weekends. So, as I start this new phase of my life, I'm looking forward to the new experiences and opportunities it brings. Beyond academics, I have a flair for cooking. I find it extremely satisfying to experiment with new recipes and techniques, often inspired by the cooking shows I love to watch. The art of preparing delicious meals truly captivates me.</p>
-<h2>Expereience</h2>
+        <nav class="navbar">
+            <a href="#">About Me</a>
+            <a href="#">Experience</a>
+            <a href="#">Skills</a>
+            <a href="#">Projects</a>
+            <a href="#">Connect</a>
+            
+        </nav>
+        
+        <div class="profile">
+<img src="profile.png" width="300" height="285"/>
+<div>
+<h3>Hello, I am</h3>
+<h1>Promesse Seuglong Fongba</h1>
+<h3>A College student, lifelong learner and an aspiring <b>Software Engineer</b></h3>
+</div>
+        </div>
+        <div class="about">
+        <h2>About Me</h2>
+<p>I have always found inspiration in the quote, "life is like a roller coaster; it has its ups and downs, but it's your choice to scream or enjoy the ride." My life's journey has indeed resembled a roller coaster, shaped by my background and socioeconomic status. I was born in Cameroon, Africa, in a family with nine kids, and I'm the seventh. Now, I'm 30 years old, a mom to two kids (four and one-year-old), and I've been married since November 2017. Even though my family didn't have a lot of money when I was growing up, I never gave up on my dream of becoming a software engineer because I love working with technology. I recently moved to the United States, embarking on a new chapter of exploration and learning. <br>Family is super important to me, and I cherish every moment I spend with them. I also really enjoy being outside, whether it's taking a walk in the park or having a picnic on the weekends. So, as I start this new phase of my life, I'm looking forward to the new experiences and opportunities it brings. Beyond academics, I have a flair for cooking. I find it extremely satisfying to experiment with new recipes and techniques, often inspired by the cooking shows I love to watch. The art of preparing delicious meals truly captivates me.</p>
+        </div>
+        <div class="experience">
+<h2>Experience</h2>
 <ul>
     <li>Customer service representative, Tanza LLC</li>
     <li>Upshifter (flexible gigs jobs), Upshift</li>
 </ul>
```

Comment:
should also add dates to the experience. In the assignment it says to use flexbox to achieve this. Could also do it the following way to align with current list items:
```
<ul class="dates">
                    <li>May 2022 - June 2023</li>
                    <li>May 2012 - June 2025</li>
</ul>

Reply:
Would have to add the following css as well: 
```
ul.dates {
  list-style: none;
}